Admin/Admin.php

```diff
@@ -697,6 +698,30 @@ public function delete($object)
     /**
      * {@inheritdoc}
      */
+    public function getVersion($object)
```

Comment:
IMO i would rename this method to something like ```getLockVersion()``` otherwise the could be some understanding problems with history/versioning